Lib/test/test_unicode.py

```diff
@@ -638,6 +638,16 @@ def test_isalpha(self):
         self.assertFalse('\U0001F40D'.isalpha())
         self.assertFalse('\U0001F46F'.isalpha())
 
+    def test_isascii(self):
+        self.assertTrue("".isascii())
+        self.assertTrue("\0".isascii())
+        self.assertTrue("\x7f".isascii())
+        self.assertFalse("\x80".isascii())
```

Comment:
I suggest to test larger code points as well:

self.assertFalse("\xe9".isascii())
self.assertFalse("\u20ac".isascii())
self.assertFalse("\U0010ffff".isascii())

My 3 favorite code points to test Latin1, BMP and non-BMP :-)

(only these tests, i don't think that it's useful to check for variant with spaces before/after)